keda/values.yaml

```diff
@@ -459,6 +459,21 @@ http:
   # -- The minimum TLS version to use for all scalers that use raw HTTP clients (some scalers use SDKs to access target services. These have built-in HTTP clients, and this value does not necessarily apply to them)
   minTlsVersion: TLS12
 
+## This setting let you enable profiling for all of the components of keda and in specific port you choose
+## This can be useful when trying to investigate errors like memory leak or cpu or even look at goroutines to understand better
+## This setting is disabled by default
+profiling:
+  keda:
+    enabled: false
+    port: 8082
+  metricsApiServer:
+    enabled: false
+    port: 8083
+  webhooks:
+    enabled: false
+    port: 8084
```

Comment:
```suggestion
profiling:
  operator:
    enabled: false
    port: 8082
  metricsServer:
    enabled: false
    port: 8083
  webhooks:
    enabled: false
    port: 8084
```

Reply:
I know this is inconsistent in the file, but let's do it this way and use `operator` and `metricsServer`

Reply:
Regarding the readme can we just reference the docs?

Reply:
I think it is better to keep the readme consistent.

by the new section I just mean the title and the rule to add `profiling` related stuff there, as it is done for other sections of the readme. Nothing too complicated

Reply:
https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#documentation

Reply:
I did add this parameters in the readme.md using the `helm-docs` command but I see it didn't add something related to the gotmpl file.
Can you maybe explain what I'm missing?

Reply:
You should update the tmpl file and then generate, see this part that adds Operations section:

https://github.com/kedacore/charts/blob/9898036f39bb9c9c479cf4ef2406678bb82a7f73/keda/README.md.gotmpl#L87C1-L95C11

The idea is to add very similar one for Troubleshooting and just filter profiling there

Reply:
Alright I understood it now

Reply:
Just changed it.
Do you want to verify? @zroubalik